curriculum/challenges/english/08-coding-interview-prep/project-euler/problem-98-anagramic-squares.english.md

```diff
@@ -7,10 +7,15 @@ forumTopicId: 302215
 
 ## Description
 <section id='description'>
-By replacing each of the letters in the word CARE with 1, 2, 9, and 6 respectively, we form a square number: 1296 = 362. What is remarkable is that, by using the same digital substitutions, the anagram, RACE, also forms a square number: 9216 = 962. We shall call CARE (and RACE) a square anagram word pair and specify further that leading zeroes are not permitted, neither may a different letter have the same digital value as another letter.
-Using words.txt (right click and 'Save Link/Target As...'), a 16K text file containing nearly two-thousand common English words, find all the square anagram word pairs (a palindromic word is NOT considered to be an anagram of itself).
+
+By replacing each of the letters in the word CARE with 1, 2, 9, and 6 respectively, we form a square number: 1296 = 36<sup>2</sup>. What is remarkable is that, by using the same digital substitutions, the anagram, RACE, also forms a square number: 9216 = 96<sup>2</sup>. We shall call CARE (and RACE) a square anagram word pair and specify further that leading zeroes are not permitted, neither may a different letter have the same digital value as another letter.
+
+Using `words`, an array of nearly two-thousand common English words, find all the square anagram word pairs (a palindromic word is NOT considered to be an anagram of itself).
```

Comment:
Again, is it worth stressing that `words` is defined for them?  I'll leave it up to you!

Reply:
I think I was trying to stick close to the original wording here, even though it's very obvious that words is defined and included in the seed. I don't mind changing the text to make it more concise. Maybe something like:

"Using the included array, find all the square anagram word pairs (a palindromic word is NOT considered to be an anagram of itself)."